### PR TITLE
pktgen_perf: disable iptables rules

### PR DESCRIPTION
**Pull request by quanwenli**
Need to disabled all rules for performance test as it affect the
performance.

ID:2196164
Signed-off-by: Wenli Quan <wquan@redhat.com>

### CODE REVIEW

**Comment by quanwenli** [comment]
JOB LOG    : /root/avocado/job-results/job-2023-05-08T03.48-1728ee6/job.log
 (1/1) 82599.bridge_test.1q.acceptance.repeat1.Host_RHEL.m9.u2.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: STARTED
 (1/1) 82599.bridge_test.1q.acceptance.repeat1.Host_RHEL.m9.u2.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.pktgen_perf.q35: PASS (187.52 s)

**Comment by quanwenli** [comment]
@yanan-fu ,could you help review this?

**Comment by yanan-fu** [comment]
It is better to configure the correct iptables rules instead of disable it totally.  More info are commented in the issue, please refer , thanks!

**Comment by quanwenli** [comment]
> It is better to configure the correct iptables rules instead of disable it totally. More info are commented in the issue, please refer , thanks!

for performance test, we need to tune the system to get better performance. so disable iptables is needed as well as other performance tests.

**Comment by yanan-fu** [comment]
> > It is better to configure the correct iptables rules instead of disable it totally. More info are commented in the issue, please refer , thanks!
> 
> for performance test, we need to tune the system to get better performance. so disable iptables is needed as well as other performance tests.

I see you point now, it is nothing related with the rules but want to disable all rules as it affect the performance, am i right ？

Could you please explain it in the commit info ? Thanks!

**Comment by quanwenli** [comment]
> > > It is better to configure the correct iptables rules instead of disable it totally. More info are commented in the issue, please refer , thanks!
> > 
> > 
> > for performance test, we need to tune the system to get better performance. so disable iptables is needed as well as other performance tests.
> 
> I see you point now, it is nothing related with the rules but want to disable all rules as it affect the performance, am i right ？
yes, you are right. 

> 
> Could you please explain it in the commit info ? Thanks!

I have added. please help review. thanks

File: generic/tests/pktgen_perf.py [hunk 1]
```diff
@@ -99,6 +100,10 @@ def _pin_vm_threads(node):
     for record in record_list.split():
         record_line += "%s|" % format_result(record, base, fbase)
 
+    if disable_iptables_rules_cmd:
+        error_context.context("disable iptables rules on host")
+        process.system_output(disable_iptables_rules_cmd, shell=True)
```
Comment (yanan-fu) [inline generic/tests/pktgen_perf.py:105]: Suggest `process.system` as no usage for the output.

**Review by yanan-fu — APPROVED** [review]
Ack

**Pull request merged**